demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -96,7 +99,12 @@ def is_docker_image_valid(self):
             error_message, error_code = Errors.non_existing_docker(
                 self.yml_docker_image
             )
-            if self.handle_error(error_message, error_code, file_path=self.file_path):
+            if self.handle_error(
+                error_message,
+                error_code,
+                file_path=self.file_path,
+                warning=not self.is_pack_xsoar_supported,
```

Comment:
why add to warning? Whats this do?

Reply:
It'll print this error as a warning.
I think that if we decide not to fail the validation for none-xsoar supported then we should at least warn about it.